packages/runtime/container-runtime/src/runningSummarizer.ts

```diff
@@ -190,16 +199,18 @@ export class RunningSummarizer implements IDisposable {
         }
     }
 
-    public handleOp(error: any, op: ISequencedDocumentMessage) {
+    public handleOp(error: any, { sequenceNumber, type, clientId, contents }: ISequencedDocumentMessage) {
         if (error !== undefined) {
             return;
         }
-        this.heuristicData.lastOpSequenceNumber = op.sequenceNumber;
+        this.heuristicData.lastOpSequenceNumber = sequenceNumber;
 
-        // Check for ops requesting summary
-        if (op.type === MessageType.Save) {
-            // TODO: cast is only required until TypeScript version 4.3
-            this.trySummarize(`save;${op.clientId}: ${op.contents}` as `save;${string}: ${string}`);
+        if (this.tryRunEnqueuedSummary()) {
+            // Check for enqueued on-demand summaries
```

Comment:
Is something supposed to happen here?

Reply:
No. I can change the comment to indicate that, or I can move it if it's confusing.

Reply:
Changed the comment to indicate that it intentionally does nothing.